src/client.ts

```diff
@@ -56,6 +57,7 @@ export class Client {
     );
     this._envDataEmitter = new InternalEventEmitter<EnvData>();
     this._registeredKeys = [];
+    this._assignedRoute = '';
```

Comment:
Typically, I'd opt for a type of `string | null` and initialize with `null` instead of an empty string.  It keeps the representation of "no data yet" uniform across types.

Reply:
make sense to me